src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20180216_instance.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="instance">
+        <createTable tableName="INSTANCE">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="clusterId" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="googleProject" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="zone" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="googleId" type="NUMERIC">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="status" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ip" type="VARCHAR(254)"/>
+            <column name="dataprocRole" type="VARCHAR(254)"/>
+            <column name="createdDate" type="TIMESTAMP(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="destroyedDate" type="TIMESTAMP(6)" defaultValue="1970-01-01 00:00:01.000000">
```

Comment:
Using the same `destroyedDate` hack that we use in the `CLUSTER` table so we can make it part of a unique index.